linux/files/modprobe.conf.jinja

```diff
@@ -0,0 +1,12 @@
+{% if module_content.disabled is defined and module_content.disabled -%}
+blacklist {{ module_name }}
+{%- else -%}
+
+{%- if module_content.option is defined -%}
```

Comment:
OK, I will change this and remove comments in this file as well than, they are not of any importance anyway.